docs/en/guides/developer/working-with-json/json-load-data.md

```diff
@@ -80,28 +80,21 @@ The `request` field from the JSON file will be stored as a tuple.
-│ timestamp │ DateTime                                                                          │              │                    │         │                  │                │
-│ clientip  │ IPv4                                                                              │              │                    │         │                  │                │
+┌─name───────┬─type──────────────────────────────────────────────────────────────────────────────┬─default_type─┬─default_expression─┬─comment─┬─codec_expression─┬─ttl_expression─┐
+│ @timestamp │ DateTime                                                                          │              │                    │         │                  │                │
+│ clientip   │ IPv4                                                                              │              │                    │         │                  │                │
 # highlight-next-line
-│ request   │ Tuple(method LowCardinality(String), path String, version LowCardinality(String)) │              │                    │         │                  │                │
-│ status    │ UInt16                                                                            │              │                    │         │                  │                │
-│ size      │ UInt32                                                                            │              │                    │         │                  │                │
-└───────────┴───────────────────────────────────────────────────────────────────────────────────┴──────────────┴────────────────────┴─────────┴──────────────────┴────────────────┘
+│ request    │ Tuple(method LowCardinality(String), path String, version LowCardinality(String)) │              │                    │         │                  │                │
+│ status     │ UInt16                                                                            │              │                    │         │                  │                │
+│ size       │ UInt32                                                                            │              │                    │         │                  │                │
+└────────────┴───────────────────────────────────────────────────────────────────────────────────┴──────────────┴────────────────────┴─────────┴──────────────────┴────────────────┘
 ```
 
 ## Insert one row
 
-When the response is inserted, all three components of the request are inserted.  The `@timestamp` field in the original dataset is also cast as a DateTime and stored in the field `timestamp`.
+When the response is inserted, all three components of the request are inserted.
 ```sql
-INSERT INTO http SELECT
-# highlight-next-line
-    toDateTime(`@timestamp`) AS timestamp,
-    clientip,
-# highlight-next-line
-    (request['method'], request['path'], request['version']),
-    status,
-    size
+INSERT INTO http SELECT *
```

Comment:
In case of `SELECT * FROM ...` we will use structure from insertion table instead of schema inference, so no need to cast timestamp and select `method`,`path` and `version` explicitly